openruth.module

```diff
@@ -433,13 +433,20 @@ function _openruth_user_status($creds = NULL, $reset = FALSE) {
  */
 function openruth_seek_reserved_item($faust_number) {
   global $user;
+
+  if (!function_exists('openruth_reservation_list')) {
+    include_once(drupal_get_path('module', 'openruth') . '/includes/openruth.reservation.inc');
+  }
```

Comment:
Thank you for elaborating @pjohans and for making the change @cableman but that does not answer the first question: "Why do we need to check if the function exist before including the file?"

Both `module_load_include` calss require_once so as I see it making this change should not be necessary at all.